app/routes.py

```diff
@@ -1,2 +1,71 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
 
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+#helper functions
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+    model = cls.query.get(model_id)
+    
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+    return model
+
+#route functions
+@planets_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet.from_dict(request_body)
```

Comment:
Doing some validation on the request body to ensure required fields are present would be nice.